Makefile

```diff
@@ -19,6 +19,12 @@ HOLESKYTEST = TESTNET=true go test -timeout 50m -v ./e2e  -parallel 4 -deploy-co
 eigenda-proxy:
 	env GO111MODULE=on GOOS=$(TARGETOS) GOARCH=$(TARGETARCH) go build -v $(LDFLAGS) -o ./bin/eigenda-proxy ./cmd/server
 
+.PHONY: eigenda-proxy-codesign
+proxy-build-sign-run:
+	env GO111MODULE=on GOOS=$(TARGETOS) GOARCH=$(TARGETARCH) go build -v $(LDFLAGS) -o ./bin/eigenda-proxy ./cmd/server
+	codesign --sign - --force --preserve-metadata=entitlements,requirements,flags,runtime ./bin/eigenda-proxy
```

Comment:
Why do you need to sign if you’re just running this locally?

Reply:
I needed to sign even if I'm running this locally, for every time I build. If this isn't needed in general, I can remove it and check my local env?

Reply:
I'm building and running this fine (both the binary and the container) without signing. Don't think we should be signing here indeed. Curious what your env config is that requires this though, once you figure it out. I've personally never signed a binary so also curious what exactly this does.

Reply:
okay interesting 🤔 I've removed this, and my env config is really simple, uses `EIGENDA_PROXY_SIGNER_PRIVATE_KEY_HEX, EIGENDA_PROXY_ETH_RPC, EIGENDA_PROXY_EIGENDA_DISPERSER_RPC, EIGENDA_PROXY_SERVICE_MANAGER_ADDR, EIGENDA_PROXY_LOG_LEVEL, EIGENDA_PROXY_LOG_PID, EIGENDA_PROXY_METRICS_ADDR, EIGENDA_PROXY_METRICS_ENABLED, EIGENDA_PROXY_METRICS_PORT`